src/types/nfts.d.ts

```diff
@@ -0,0 +1,24 @@
+type TNFTBalance = {
+  chainId: number;
+  collectionAddress: string;
+  collectionName: string;
+  collectionImage: string;
+  floorPrice: number;
+  totalCount: number;
+  totalValue: number; // USD
+};
+
+type TNFTActivityV2 = {
```

Comment:
```suggestion
type TNftTransferActivityV2 = {
```

To keep it consistent with the type in Evmscan